features/projects/src/main/kotlin/com/agilefreaks/freaks_catalog/features/projects/ProjectsApolloClient.kt

```diff
@@ -0,0 +1,7 @@
+package com.agilefreaks.freaks_catalog.features.projects
```

Comment:
From what I remember this is added in the project, why do we need to create another apolloclient?